src/Autofac/ResolutionExtensions.cs

```diff
@@ -55,7 +55,22 @@ public static class ResolutionExtensions
         /// <returns><paramref name="instance"/>.</returns>
         public static TService InjectProperties<TService>(this IComponentContext context, TService instance)
         {
-            AutowiringPropertyInjector.InjectProperties(context, instance, true);
+            AutowiringPropertyInjector.InjectProperties(context, instance, DefaultPropertySelector.OverwriteSetValueInstance);
+            return instance;
+        }
+
+        /// <summary>
+        /// Set any null-valued properties on <paramref name="instance"/> that can be
```

Comment:
I think the summary here may be a copy/paste error. It uses the selector, not null-value properties.